lib/support/nginx/huginn

```diff
@@ -0,0 +1,70 @@
+## Huginn
```

Comment:
I think the doc directory should just contain documentation I got `lib/support` from GitLab, we still have the `deployment` directory should I move the files there?